collector/nodes.go

```diff
@@ -1781,6 +1789,176 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
 				Labels: defaultFilesystemIODeviceLabelValues,
 			},
 		},
+		indexingPressureMetrics: []*indexingPressureMetric{
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "indexing_pressure", "current_combined_coordinating_and_primary_in_bytes"),
```

Comment:
This seems like a composite metric that is the sume of the `current_coordinating_in_bytes` and `current_primary_in_bytes` metrics. We typically drop these things to save space and cardinality when you can use `foo + bar` in PromQL.

Reply:
thanks for your review.  After I check the document, `current_combined_coordinating_and_primary_in_bytes` or `total_combined_coordinating_and_primary_in_bytes` is not equal to `coordinating + primary`.    So maybe we should keep this?    Here is the document:
```
# https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html#cluster-nodes-stats-api-response-body-indexing-pressure
combined_coordinating_and_primary_in_bytes: 
(integer) Memory consumed, in bytes, by indexing requests in the coordinating or primary stage. This value is not the sum of coordinating and primary as a node can reuse the coordinating memory if the primary stage is executed locally.
```



Please let me known if you have any questions or concern.